packages/typespec-client-generator-core/src/package.ts

```diff
@@ -175,13 +175,15 @@ function createContentTypeOrAcceptHeader(
     };
   }
   // No need for clientDefaultValue because it's a constant, it only has one value
+  // Since these are the created accept and content type header parameters, we don't make them required.
```

Comment:
What's the motivation for this change? Since these are synthesized, constant, params, they won't show up in public surface area so there's no way for callers to set them. Maybe I'm missing something.